index.html

```diff
@@ -0,0 +1,738 @@
+        
+                                <ul class="details">
+                                    <li>by Napier Lopez</li>
+                                    <li><i class="far fa-clock"></i> 26 minutes ago</li>
+                                    <li><i class="fas fa-share-alt"></i> 5</li>
+                                </ul>
+                        </div>
+                       
+                    </div>
+                    <div class="second-third-articles">
+                        
+                            <div class="article2">
+                                    <div class="overlay2"></div>
+                                    <img class="article-main-image" src="images/security-cam.jpg" alt="security camera">
+
+                                <div class="text">
+                                    <span class="category">APPS</span>
+                                    <h2 class="second-title"><a class="title-link" href="#">Bought a used Nest security cam? The previous owner can spy on you</a></h2>
+                                    
+                                    <ul class="details">
+                                        <li> by Rachel Kaser</li>
+                                        <li> <i class="far fa-clock"></i> 1 hour ago</li>
+                                        <li><i class="fas fa-share-alt"></i> 0</li>
+                                    </ul>
+                                </div>
+                            </div>
+                        
+                            <div class="article3">
+                                    <div class="overlay2"></div>
+
+                                    <img class="article-main-image" src="images/small-crypto.png" alt="PWC new cryptocurrency">
+
+                                <div class="text">
+                                    <span class="category">TECH</span>
+                                    <h2 class="second-title"><a href="#">PwC has a new cryptocurrency auditing tool… and actually, it sounds alright</a></h2>
+                                    
+                                    <ul class="details">
+                                        <li> by Yaron Yitzhak</li>
+                                        <li> <i class="far fa-clock"></i> 4 hours ago</li>
+                                        <li><i class="fas fa-share-alt"></i> 1</li>
+                                    </ul>
+                                </div>
+                            </div>
+                    
+                    </div>
+                
+                   
+                </section>
+            
+        <!-- LATEST NEWS -->
+
+        <h3 class="latest-news-title"><a href="#">Latest news <i class="fas fa-greater-than"></i></a></h3>
+
+        <div class="latest-news">
+
+
+                    <div class="latest-news-thumbnail">
```

Comment:
This should be an `article` tag instead. Each one of the 8 latest news articles must be inside an `article` tag. Kindly do that with them all.